<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageProgrammesDialog</class>
 <widget class="QDialog" name="MessageProgrammesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Messages programmés</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="messageTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="supprimerAllButton">
       <property name="text">
        <string>Supprimer tout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="supprimerButton">
       <property name="text">
        <string>Supprimer la sélection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifierButton">
       <property name="text">
        <string>Modifier la sélection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addMessageButton">
       <property name="text">
        <string>Ajouter une nouvelle programmation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageProgrammesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
